mobile-app/app/screens/AppNavigator/screens/Dex/components/ButtonGroup.tsx

```diff
@@ -49,15 +57,29 @@ interface ButtonGroupItemProps {
   width: BigNumber
   testID: string
   modalStyle?: StyleProp<TextStyle>
+  portfolioButtonGroupStyle?: { [key: string]: string }
+  portfolioLightActiveStyle?: boolean
```

Comment:
- remove `portfolioLightActiveStyle ` and `portfoliolDarkActiveStyle`
- replace with `customActiveStyle` of type `ThemedProps?`

Reason:
- to support future parent component that also wants to have custom active styling